### PR TITLE
add banners for fin template markets

### LINKED ISSUE #4514: Market creation: banner for finance templates
If a user chooses a stock/index template, they need to enter either: 
- the stocks ticker symbol
or
- the indexes full name
in the market question to be considered valid. 

We'll highlight this to users by displaying a banner below the Market Question on these forms. 

Stock: 
https://www.figma.com/file/IpIAYVQPeQfuT5TTUboQpo4S/Market-creation?node-id=2771%3A9222

`Enter stock ticker symbol to ensure the market resolves as valid, i.e. AAPL`

image

Index:
https://www.figma.com/file/IpIAYVQPeQfuT5TTUboQpo4S/Market-creation?node-id=2807%3A655

`Enter the full name of the index to ensure the market resolves as valid, i.e. S & P 500 Index`

image

### PR DESCRIPTION
**Pull request by bthaile**
closes <https://github.com/augurproject/augur/issues/4514>

only shows up if market is in INDEXES or STOCKS category. easy to verify using a financial template

image

image

custom market creation
image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🖖

 This link will always point to this PR's latest changes: https://peek.run/vv2h4pg

File: packages/augur-ui/src/modules/create-market/components/common.tsx [hunk 1]
```diff
@@ -47,6 +47,8 @@ import {
   CHOICE,
   REQUIRED,
 } from '@augurproject/artifacts';
+import { TemplateBannerText } from '../constants';
```
Comment (andrewdamelio) [inline packages/augur-ui/src/modules/create-market/components/common.tsx]: relative-path detected ```'../constants'```

**Comment by andrewdamelio** [comment]
@bthaile I didn't see the banner for the third case - custom market with categories set to FIN/Stock

**Comment by bthaile** [comment]
testing again, all code has been merged up

**Comment by bthaile** [comment]
image

**Comment by bthaile** [comment]
there's a conflict that's the problem.

**Comment by bthaile** [comment]
resolved conflict, try it again

File: packages/augur-ui/src/modules/create-market/components/common.tsx (hunk 1 above)
Comment (bthaile) [reply packages/augur-ui/src/modules/create-market/components/common.tsx]: doh!!!! pushed fix

**Comment by bthaile** [comment]
also works for custom inputted values

image

**Review by andrewdamelio — APPROVED** [review]

**Pull request merged**